apisix/plugins/ext-plugin/init.lua

```diff
@@ -677,6 +677,7 @@ local function create_lrucache()
 
     lrucache = core.lrucache.new({
         type = "plugin",
+    	invalid_stale = true,
```

Comment:
Could you do some refactor so we don't need to specify the lrucache options twice? Thanks!